def/babel6.js

```diff
@@ -0,0 +1,117 @@
+module.exports = function (fork) {
+    fork.use(require("./babel"));
+    fork.use(require("./flow"));
```

Comment:
I think you should fork both. It's conceivable people will want to use the types defined in `./babel` without those defined in `./flow`, right? If `./babel` forks `./flow`, it should imply that those type definitions somehow depend on `./flow`.